.gitignore

```diff
@@ -1,3 +1,92 @@
-/node_modules
+# dependencies
+node_modules
```

Comment:
Is that node_modules rule contains node_modules in client directory?

Reply:
Yes, for me add both to ignore

Reply:
Approved